doc-content/enterprise-only/openshift/configuring-image-registry-images-proc.adoc

```diff
@@ -0,0 +1,59 @@
+[id='configuring-image-registry-images-proc_{context}']
+= Configuring the image registry to use specific images
+
+During the basic configuration of the environment you can configure the custom registry to use specific images. For more information about configuring the basic environment, see <<operator-deploy-basic-proc_{context}>>.
+
+.Prerequisites
+* You started to configure a {PRODUCT} environment using {operator_name} and accessed the installer wizard according to the instructions in <<operator-deploy-basic-proc_{context}>>.
+* You have set the URL of the image registry in the *Image registry* field under the *Custom registry* of the {PRODUCT} environment configuration according to the instructions in <<operator-deploy-basic-proc_{context}>>.
+
+.Procedure
+
+. During the {PRODUCT} environment configuration, to use a specific image from the custom registry, perform one of the following tasks:
+* If you want to specify the image using the installer wizard, set the image context, image name, and image tag parameters in the *Console* and *KIE Server* tabs:
+
+.Parameters
+[cols="25%,75%", frame="all", options="header"]
+|===
+|Name
+|Description
+|*Image context*
+|The context of the image in the registry.
+|*Image*
+|The name of the image.
+|*Image tag*
+|The tag of the image. If you do not set this field, the installation uses the latest tag.
+|===
+
+* You want to specify the image by using a KieApp CR YAML file, add the the image registry and image details to the file, for example:
++
+.Example
+[source,yaml]
+----
+apiVersion: app.kiegroup.org/v2
+kind: KieApp
+spec:
+ ...
+ useImageTags: true
+ imageRegistry:
+   registry: registry.example.com:5000
+   ...
+ objects:
+   ...
+   servers:
+     - id:  ...
+       ...
+       image: YOUR_IMAGE_NAME
+       imageContext: YOUR_IMAGE_CONTEXT
+       imageTag: YOUR_IMAGE_TAG
+       ...
+----
++
+[NOTE]
+====
+The operator does not create an Imagestream if there is an existing imagestream in the namespace.
```

Comment:
I would suggest making this phrase more positive. Something like:

If an ImageStream with the configured name exists in the namespace that you are using or in the OpenShift namespace, the operator uses this ImageStream and does not create a new ImageStream.